src/v2/Apps/Conversation/ConversationApp.tsx

```diff
@@ -31,7 +31,10 @@ const getViewWidth = () => {
 const Inbox: React.FC<InboxProps> = ({ selectedConversation, me }) => {
   return (
     <>
-      <ConversationList me={me} />
+      <ConversationList
+        me={me}
+        selectedConversationID={selectedConversation.internalID}
```

Comment:
This feels like it might be worth double checking with someone on Purchase, this _might_ alter the UI in some way (a quick glance in `ConversationList` reveals the prop is used).

Reply:
The prop is required by `ConversationList` so I would assume we want it there, @zephraph do you have any context on this?

Reply:
Oh! Seems like you might've caught a legit bug 😄. I'm 👍 with this change. That's what's supposed to be happening (but the visual distinction is admittedly pretty minor so a bit easy to miss).